CMP1903_A1_2324/Program.cs

```diff
@@ -4,7 +4,31 @@ namespace Dice2
 
 class Program
 {
-        static void Main(string[] args);
+    static void Main(string[] args)
 
-   
+    {
+        try
+        {
+            //RunTests();
+            Game Dice = new Game();
+            Dice.Roll();
+        }
+        /*catch(InvalidDiceException e)
```

Comment:
To make it easier to read you could remove this as it is unnecessary and could cause confusion.